proxy/proxy/client.go

```diff
@@ -32,14 +32,14 @@ import (
 const (
 	DefaultRefreshCfgThrottle = time.Minute
 	keepAlivePeriod           = time.Minute
+	defaultRefreshCertBuffer  = 2 * time.Minute
```

Comment:
I know originally it was 30 seconds, but I think bumping this value to 5 minutes would be an improvement - with a 1 min refresh that would be ~4-5 retries before the cert expires